app/metrics.py

```diff
@@ -0,0 +1,31 @@
+from flask import Blueprint
+from flask.signals import got_request_exception, request_finished
+
+from gds_metrics import GDSMetrics
+
+
+metrics = Blueprint('metrics', __name__)
+
+
+class DMGDSMetrics(GDSMetrics):
```

Comment:
Didn't we move this to utils? Am I getting confused?

Reply:
Good shout, copied the wrong file